core/src/main/java/tachyon/client/TachyonFS.java

```diff
@@ -132,13 +133,12 @@ public static synchronized TachyonFS get(String masterHost, int masterPort, bool
 
   // All Blocks has been locked.
   private final Map<Long, Set<Integer>> mLockedBlockIds = new HashMap<Long, Set<Integer>>();
+  // Mapping from block id to path of the block locked
+  private final Map<Long, String> mLockedBlockIdToPath = new HashMap<Long, String>();
```

Comment:
Why have this? This data should be local to the methods that need it. Between requests sever may move the block